index.html

```diff
@@ -103,6 +117,7 @@
           Start Game
         </mdl-button>
         <div id="main-view">
+           <papan-user-profile id="profile-view" style="display: none"/>
         </div>
       </mdl-content>
     </mdl-layout>
```

Comment:
Nicolas just told me about databinding, so I'm looking at it. I'm new to all these frameworks. but it seems like a good idea.